1001-2000/LIT1171Awdana.xml

```diff
@@ -53,15 +64,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <ptr target="bm:ContiRossini1941AwdaNagast"/>
                </bibl>
             </listBibl>
+            <listBibl type="secondary">
+               <bibl> 
+                  <ptr target="bm:Pietruschka2003Awda"/>
+               </bibl>
+            </listBibl>
          </div>
-        <div type="edition">
-           <div type="textpart" xml:id="tables">
+         <div type="edition">
+            <div type="textpart" xml:id="tables">
                <label>Sixteen circular tables, each subdivided into sixteen parts with questions on various topics</label>
-           </div>
+            </div>
             <div type="textpart" xml:id="seas">
                <label>Sixteen "seas" (<foreign xml:lang="gez">ባሕር፡</foreign>), each containing 16 sentences as a response to the previous questions</label>
+               <note>The following incipit and explicit are taken from <ref type="mss" corresp="BAVet119"/>.</note>
+               <div type="textpart" subtype="incipit" xml:lang="gez">
+                  <ab>
+                     በስመ፡ አብ፡ <supplied reason="ellipsis"/> ፩ አምላክ፡ መሐሪ፡ ወመስተሣህል፡ ዘተናገረ፡ ፩ጠቢብ፡ እምነ፡ ብዙኃን፡ ጠቢባን፡ ወፈላስፋውያን፡
+                     ዘንተ፡ ዘስሙ፡ መጽሐፈ፡ ምክር፡ ለአበው፡  <sic>ቀደመት፡</sic> ከመ፡ ያእምሩ፡ በቱ፡ በዝ፡ እምሂ፡ ሠናየ፡ ወዕመሂ፡ እኩየ፡
+                     ሰብአ፡ ዓለም፡ ለለርእሶሙ፡ ውእቱ፡ ፩ በኃሢሠ፡ መፍቅድ፡ ሐቲት፡ ፪ ሐቲት፡ በነጊደ፡ ንዋይ፡ ፫ ሐቲት፡ 
+                     በበዊዓ፡ ቤተ፡ ንጉሥ፡ 
+                     ፩ በኃሢሠ፡ መፍቅድ፡ ናሁ፡ መጽአ፡ ለከ፡ ሠናይ፡ ዘያስተፌሥሕ፡ ልበከ፡ ብስራት፡ ለከ፡ እስመ፡ ይትፌጸም፡ ጻሕቅከ፡
+                  </ab>
+               </div>
+               <div type="textpart" subtype="explicit" xml:lang="gez">
+                  <ab>
+                     ፲፮ ሐቲት፡ በተዋስቦ፡ እመሰ፡ ትፈቅድ፡ ተዋስቦ፡ ተዓገሥ፡ እስመ፡ <sic>በተዓገሦ፡</sic> ይትረከብ፡
+                     በትፍሥሕት፡ <sic>ወኃሢት፡</sic> ወብዕል፡ ወኵሉ፡ <sic>ትምኒትከ፡</sic> ወአንተ፡ <sic>ኵን፡</sic> ኅዱዓ፡
+                     በኵሉ፡ ነገረ፡ ተዋስቦ፡ ወዝኒ፡ ተዋስቦ፡ ዘትሄሊ፡ ይከውኖሙ፡ ለእሉ፡ ትፍሥሕት፡ <sic>ወኃሢት፡</sic> ወብዕል፨
```

Comment:
Maybe mark that ነገረ፡ is a correction by Grébaut-Tisserant (as you do in the next description)